tbans/models/messages/webhook.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from consts.notification_type import NotificationType
+
+
+# Represents a webhook notification object
+class WebhookMessage:
+
+    def __init__(self, notification):
+        self.notification = notification
+
+    # TODO: repr, so we can log
+
+    """
+    JSON for WebhookMessage will look like...
+    {
+        'message_data': {...},
+        'message_type': ...
+    }
+    """
+    def json_string(self):
+        json_dict = {
+            'message_type': NotificationType.type_names[type(self.notification)._type()]
```

Comment:
Design question - it seems like we should send this `message_type` automatically in the data payloads for push notifications, right? That's probably going to be our only way to identify what type of notification is coming from the server, and how to respond to it in clients.